docs/src/modules/components/HomeSteps.js

```diff
@@ -205,6 +205,9 @@ function HomeSteps(props) {
                 className={classes.img}
                 alt="themes"
                 src={`/static/images/themes-${theme.palette.type}.jpg`}
+                loading="eager"
+                width={500}
```

Comment:
I believe we override the dimension with CSS, do we need to specify something here?

Reply:
The browser needs to to calculate a proper placeholder and avoid layout shifts.